src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts

```diff
@@ -278,12 +281,19 @@ export class SignupComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
   resolved(captchaResponse: string) {
-    const newResponse = captchaResponse
-      ? `${captchaResponse.substr(0, 7)}...${captchaResponse.substr(-7)}`
-      : captchaResponse;
-    this.captchaResponse += `${JSON.stringify(newResponse)}\n`;
-    if (this.captchaResponse) {
-      this.onSubmitSignUpForm();
+    if (captchaResponse) {
```

Comment:
Create the recaptchaService for this. So that we can use multiple places the same API.